build.py

```diff
@@ -912,8 +928,6 @@ def container_build(images, backends, repoagents, endpoints):
         base_image = images['base']
     elif target_platform() == 'windows':
         base_image = 'mcr.microsoft.com/dotnet/framework/sdk:4.8'
-    elif target_platform() == 'ubuntu/arm64':
```

Comment:
Don't we want the aarch64 base image here when platform is ARM?

Reply:
The base image is now a manifest that points to both the amd64 and arm64 images. Docker will pull the appropriate one based on the host platform